### DISCUSSION TITLE
Shortcut (formerly Clubhouse) integration

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5458] [New features]
We've had approx. 4 requests here. 

User requests: #3210

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/52486451) 🔒 writes:

> Our organization and multiple other small companies use Shortcut for issue tracking. It would be awesome to have this integration.

**Comment by brentchow** [comment]
+1 😊
